systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -144,6 +155,48 @@ protected void installClusterOperator(String namespace) throws Exception {
         installClusterOperator(namespace, Constants.CO_OPERATION_TIMEOUT_DEFAULT);
     }
 
+    /**
+     * Perform application of ServiceAccount, Roles and CRDs needed for proper cluster operator deployment.
+     * Configuration files are loaded from install/cluster-operator directory.
+     */
+    public void applyClusterOperatorInstallFiles(String namespace) {
+        clusterOperatorConfigs.clear();
+        Map<File, String> operatorFiles = Arrays.stream(new File(CO_INSTALL_DIR).listFiles()).sorted()
+                .filter(file ->
+                        !file.getName().matches(".*(Binding|Deployment)-.*"))
+                .collect(Collectors.toMap(
+                    file -> file,
+                    f -> TestUtils.getContent(f, TestUtils::toYamlString),
+                    (x, y) -> x, LinkedHashMap::new));
+        for (Map.Entry<File, String> entry : operatorFiles.entrySet()) {
+            LOGGER.info("Applying configuration file: {}", entry.getKey());
+            if (Environment.isRolesOnly()) {
+                switchClusterRolesToRoles(entry.getValue());
+            }
+            clusterOperatorConfigs.push(entry.getKey().getPath());
+            cmdKubeClient().clientWithAdmin().namespace(namespace).apply(entry.getKey().getPath());
+        }
+    }
+
+    /**
+     * Replace all references to ClusterRole to Role.
+     * This includes ClusterRoles themselves as well as RoleBindings that reference them.
+     */
+    private void switchClusterRolesToRoles(String fileContents) {
+        fileContents.replace("ClusterRole", "Role");
+    }
```

Comment:
I guess you will need to also fix the ClusterRoleBindings somewhere? To not install them randomly against missing clusterRoles?